### PR TITLE
TACO-449 Add a new version of IIQ script behind a flag

### PR DESCRIPTION
**Pull request by bckatarzyna**

### CODE REVIEW

**Review by szymon-nawrocki — APPROVED** [review]

File: @types/intent-iq/index.d.ts [hunk 1]
```diff
@@ -26,7 +26,8 @@ interface IntentIQConfig {
 	partnerClientId?: string;
 	partnerClientIdType?: number;
 	callback?: (data: IntentIqResponseData) => void;
-	browserBlackList?: string;
+	browserBlackList?: string[];
```
Comment (jbienkowski-fandom) [inline @types/intent-iq/index.d.ts:29]:
```suggestion
        // TODO: remove type definition after new IIQ script is fully released
	browserBlackList?: string;
```
I guess this should remain a string?

File: src/ad-bidders/prebid/intent-iq/intent-iq.ts [hunk 2]
```diff
@@ -55,16 +60,27 @@ export class IntentIQ {
...
+				pbjs,
+				timeoutInMillis: DEFAULT_MAX_DELAY,
+				ABTestingConfigurationSource: 'percentage',
+				abPercentage: 97,
+				manualWinReportEnabled: true,
+				domainName,
+			};
+
+			if (this.intentIQVersion === newScriptVersion) {
+				intentIqConfig['vrBrowserBlackList'] = ['chrome'];
```
Comment (jbienkowski-fandom) [inline src/ad-bidders/prebid/intent-iq/intent-iq.ts:74]: (nit) 
```suggestion
				intentIqConfig.vrBrowserBlackList = ['chrome'];
```

File: src/ad-bidders/prebid/intent-iq/intent-iq.ts [hunk 3]
```diff
@@ -55,16 +60,27 @@ export class IntentIQ {
...
+			} else {
+				intentIqConfig['browserBlackList'] = 'Chrome';
```
Comment (jbienkowski-fandom) [inline src/ad-bidders/prebid/intent-iq/intent-iq.ts:76]: (nit) 
```suggestion
				intentIqConfig.browserBlackList = 'Chrome';
```
I'm wondering why linter did not pick this up 🤔

File: src/ad-bidders/prebid/intent-iq/intent-iq.ts [hunk 4]
```diff
@@ -55,16 +60,27 @@ export class IntentIQ {
...
 				: window.location.hostname;
 
+			const intentIqConfig = {
+				partner: this.fandomId,
+				pbjs,
+				timeoutInMillis: DEFAULT_MAX_DELAY,
+				ABTestingConfigurationSource: 'percentage',
+				abPercentage: 97,
+				manualWinReportEnabled: true,
+				domainName,
```
Comment (jbienkowski-fandom) [inline src/ad-bidders/prebid/intent-iq/intent-iq.ts:70]:
```suggestion
				domainName,
				...(this.intentIQVersion === newScriptVersion ? { vrBrowserBlackList: ['chrome'] } : { browserBlackList: 'Chrome' } ),
```
(nit) alternatively, kinda Reactish way

Comment (bckatarzyna) [reply src/ad-bidders/prebid/intent-iq/intent-iq.ts:70]: Nice, thank you! 💎

**Review by jbienkowski-fandom — APPROVED** [review]

**Pull request merged**